package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "k-intl",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "main": "index.js",
   "author": "Guillaume CRESPEL <guillaume.crespel@gmail.com>",
   "license": "MIT",
   "scripts": {
+    "build:data": "babel-node scripts/build-data",
     "build:core": "cross-env NODE_ENV=build rollup -c ./misc/rollup.config.js",
-    "build": "npm-run-all --parallel build:*",
+    "build": "npm-run-all build:*",
```

Comment:
Why do you remove parallel option ?

Reply:
because build:data make src/en.js (the default language). it must be the first action.

Reply:
You can make `en.js` external maybe ?

Reply:
@guillaumecrespel and about that ?